dsa-23au/PeriodicTable.md

```diff
@@ -0,0 +1,18 @@
+# Periodic Table of DSA
+
+| Data Structure Name | Create                        || Read                                             ||||| Update         | Delete      |
+|---------------------|---------------|---------------|--------|-----------|--------------|-------|----------|----------------|-------------|
+| From size n, afterwards...  | size increases to n+1 || size stays the same at `n`   ||||| size stays the same at `n`         | size decreases to `n-1`      |
+|                     | insertAt(i,v) | insertLast(v) | size() | isEmpty() | searchFor(v) | min() | max()    | replaceAt(i,v) | removeAt(i) |
+| Arrays              | [O(?)](#arrays-insert-at)          |               |        |           |              |       |          |                |             |
+| Linked lists        |               |               |        |           |              |       |          |                |             |
+| Stacks              |               |               |        |           |              |       |          |                |             |
+| Queues              |               |               |        |           |              |       |          |                |             |
+| Heaps               |               |               |        |           |              |       |          |                |             |
+| Binary Search Trees |               |               |        |           |              |       |          |                |             |
+
+## Arrays
+
```

Comment:
I'm a little confused about what we're supposed to put in the table.

Reply:
Your goal is to fill in an entry, in between two of the `|` characters, for a given row (data structure) and a given column (operation or algorithm), using Big-Oh notation, to say how long that operation takes on that data structure.

Here are some examples:
https://github.com/TheEvergreenStateCollege/upper-division-cs/commits/main/dsa-23au/PeriodicTable.md